frontend/js/Cart.js

```diff
@@ -36,14 +37,15 @@ function addItemToCart(item) {
                     };
                 });
             } else {
-                // L'article n'est PAS présent dans le panier, on l'y ajoute
+                /** L'article n'est PAS présent dans le panier, on l'y ajoute */
                 cart.push(item);
                 localStorage.setItem("Panier", JSON.stringify(cart));
                 numberOfItemsInCart();
             }
     } else {
-        // Si le panier n'est pas présent au sein du localStorage
-        // On créé le panier et y ajoute le nouvel article désiré avec toutes ses caractéristiques
+        /** Si le panier n'est pas présent au sein du localStorage :
+         *   on créé le panier et y ajoute le nouvel article désiré avec toutes ses caractéristiques
+         */
```

Comment:
on mets pas de commentaire sur la ligne /**

Reply:
Vu que c'est sur une ligne il me semblait que c'était ok comme ça ?

/** This is a description of the foo function. */
function foo() {
}
> vu sur la doc : https://jsdoc.app/about-getting-started.html